x-pack/plugins/monitoring/server/alerts/missing_monitoring_data_rule.ts

```diff
@@ -59,20 +58,21 @@ export class MissingMonitoringDataRule extends BaseRule {
   protected async fetchData(
     params: CommonAlertParams,
     esClient: ElasticsearchClient,
-    clusters: AlertCluster[],
-    availableCcs: boolean
+    clusters: AlertCluster[]
   ): Promise<AlertData[]> {
-    let indexPattern = appendMetricbeatIndex(Globals.app.config, INDEX_PATTERN);
-    if (availableCcs) {
-      indexPattern = getCcsIndexPattern(indexPattern, availableCcs);
-    }
+    // changing this to only search ES because of changes related to https://github.com/elastic/kibana/issues/83309
```

Comment:
changes were made in `fetchMissingMonitoringData` that only returns ES data, so I changed the index pattern here to only search `.monitoring-es` instead of `.monitoring-*`.  https://github.com/neptunian/kibana/commit/ced1dadb8a5c2dcf50f590dbe3c5db38d512e0c1#diff-666dd698bc0e64e0a8265f7f08b758b5d73921ea321e0312f32474e3f834ac9bR40